packages/x-components/src/x-modules/facets/store/module.ts

```diff
@@ -17,10 +18,14 @@ export const facetsXStoreModule: FacetsXStoreModule = {
     groups: {},
     facets: {},
     preselectedFilters: [],
-    stickyFilters: {}
+    stickyFilters: {},
+    config: {
+      filtersForRequestStrategy: 'all'
```

Comment:
Shouldn't we default to the `leaves-only`? 🤔

Reply:
Default behavior for the library is `all` and for our implementation is `leaves-only`.